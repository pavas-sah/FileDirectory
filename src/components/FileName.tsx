import React from 'react';

export interface FileNameProps {
  fileName: string;
}

export const FileName = (props: FileNameProps) => {
  return <>{props.fileName}</>;
};
export default FileName;
