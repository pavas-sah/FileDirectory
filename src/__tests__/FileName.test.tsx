import React from 'react';
import renderer from 'react-test-renderer';
import FileName, { FileNameProps } from '../components/FileName';

const NameProps: FileNameProps = {
  fileName: 'someFile',
};

describe('File Name Functional component', () => {
  it('should have some message as content.', () => {
    const fileNameComponent = renderer.create(<FileName {...NameProps} />);
    expect(fileNameComponent.root.children.toString()).toBe('someFile');
  });

  it('matches snapshot', () => {
    const tree = renderer.create(<FileName {...NameProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
