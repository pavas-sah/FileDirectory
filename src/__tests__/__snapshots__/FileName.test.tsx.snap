// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File Name Functional component matches snapshot 1`] = `"someFile"`;
